src/Nethermind/Nethermind.State/StateProvider.cs

```diff
@@ -456,7 +456,7 @@ public ChangeTrace(Account? after)
             public Account? After { get; }
         }
 
-        public void Commit(IReleaseSpec releaseSpec, IStateTracer stateTracer)
+        public void Commit(IReleaseSpec releaseSpec, IStateTracer stateTracer, bool isGenesis = false)
```

Comment:
hmmmm, I think we should discuss it with the test team and they should fix it?

Reply:
or simply disable EIP158 in Hive during the genesis commit?

Reply:
Ok. thx. You're right. I thought about that. On the other hand, we can treat the genesis file as something that we shouldn't modify, so it has the sense to skip removing accounts from the genesis. However, I'll discuss it.

Reply:
I would disable through eip if possible